<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TasksWindow</class>
 <widget class="QDialog" name="TasksWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="TasksFunctionsBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>680</width>
     <height>450</height>
    </rect>
   </property>
   <property name="title">
    <string>Tasks Functions</string>
   </property>
   <widget class="QGroupBox" name="CreateTaskBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Create Task</string>
    </property>
    <widget class="QPushButton" name="CreateTaskButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Create Task</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="UpdateTaskBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Update Task</string>
    </property>
    <widget class="QPushButton" name="UpdateTaskButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Update Task</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="DeleteTaskBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Delete Task</string>
    </property>
    <widget class="QPushButton" name="DeleteTaskButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete Task</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="EventsBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>510</width>
      <height>170</height>
     </rect>
    </property>
    <property name="title">
     <string>Events</string>
    </property>
    <widget class="QTableView" name="EventsTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>490</width>
       <height>140</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="TasksForTheEventBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>510</width>
      <height>170</height>
     </rect>
    </property>
    <property name="title">
     <string>Tasks for the Event</string>
    </property>
    <widget class="QTableView" name="TasksForTheEventTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>490</width>
       <height>140</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>641</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="HorizontalLayout">
     <item>
      <spacer name="HorizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
